src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1467,6 +1484,12 @@ class IngestUploadCmd(Command):
 
     def execute(my):
 
+        FOLDER_LIMIT = 500
```

Comment:
define a constant under the class instead and refer to it as my.FOLDER_LIMIT later on